src/components/ShareURIDialog.vue

```diff
@@ -22,6 +22,11 @@
               {{ $t('home.show_qr_code') }}
             </v-list-item-title>
           </v-list-item>
+          <v-list-item v-if="!isErc" @click="openInExplorer">
```

Comment:
ERC20 tokens also have an address (same as for ETH).
Handle that logic in the `openInExplorer` method.